Sources/Swift/Integrations/SessionReplay/SentryReplayBreadcrumbConverter.swift

```diff
@@ -1,8 +1,13 @@
 @_implementationOnly import _SentryPrivate
 import Foundation
 
+@objc(SentryReplayBreadcrumbConverter)
+protocol SentryReplayBreadcrumbConverterProtocol: NSObjectProtocol {
+    func convert(breadcrumbs: [Breadcrumb], from: Date, until: Date) -> [SentryRRWebEvent]
+}
+
 @objcMembers
-class SentryReplayBreadcrumbConverter: NSObject {
+class SentryReplayBreadcrumbConverter: NSObject, SentryReplayBreadcrumbConverterProtocol {
```

Comment:
This base converter should also be exposed to hybrid SDKs so that we can delegate to it after custom processing is done. Like here: https://github.com/getsentry/sentry-react-native/pull/3846/files#diff-e405eb323a493cbd718d41b79f2642874fc60ff229e17b139f8516b07fe2a7a2R35

Reply:
What's your plan of exposing the base converter to RN? We cannot subclass the swift class from obj-c

Reply:
https://github.com/getsentry/sentry-react-native/pull/3846/files#diff-0a664422c5522958ee625f390b58d6b833bab7d360e761b371a3815c827e206dR25